index.js

```diff
@@ -144,6 +144,86 @@ function reset () {
   moduleAliasNames = []
 }
 
+//
+// Utility function
+//
+
+// Removes file name from the base and returns only dir.
+function getBaseDir (base) {
+  var parsed = nodePath.parse(base)
```

Comment:
Unfortunately "parse" was added in node 0.11 ... And we support up to 0.10. @ilearnio Should we drop support for 0.10 in major version change to 3.x ?

Otherwise you have to hack around it, with `path.extname`, `path.dirname` and potentially `fs.lstatsync`.

Reply:
Can be replaced with 

```
var isFile = nodePath.extname(base)
return isFile ? nodePath.dirname(base) : base
```

Not sure how `fs.lstatsync` can be used.

Reply:
It works in all cases except when the config file is called "my-config", without extension.

To know for sure whether a file is a file or a directory, you can call `fs.lstatSync('path/to/file').isFile()`

Reply:
Wow, this is great. Will update it.